.github/dependabot.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+updates:
+  # === Root package.json (/) ===
+  # Internal dependencies
+  - package-ecosystem: 'npm'
+    directory: '/'
```

Comment:
can't you use `directories` instead and remove the duplicate?

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#directories-or-directory--